src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/custom.py

```diff
@@ -287,7 +287,7 @@ def list_ip_addresses(resource_group_name=None, vm_name=None):
 
 
 def create_managed_disk(resource_group_name, disk_name, location=None,
-                        size_gb=None, sku=None,
+                        size_gb=None, sku='Premium_LRS',
```

Comment:
Is "None" actually a valid size?  Does the server have defaulting logic, or should this be required?